### ISSUE TITLE
Plugin wrongly adds JavaSE-8 entry java.configuration.runtimes in settings.json

### ISSUE THREAD

**Issue by RAIMUNDOS54** [graalvm/vscode-extensions#21]
This entry is added by the plugin. It can't be found. Even if the right entry is already added, which is JavaSE-1.8, the problem persists.

    {
      "name": "JavaSE-8",
      "path": "C:\\Program Files\\GraalVM\\graalvm-ee-java8-21.3.4"
    }

Thank you.

**Comment by MartinBalin** [comment]
the problem is that "name" should be JavaSE-1.8 instead of 8, right?

**Comment by RAIMUNDOS54** [comment]
That's correct. But as I see, the name JavaSE-1.8 is already there. I just don't know if the extension adds only the SE-8, only the SE-1.8 or both since I used the Red Hat and Microsoft Java extensions for some time. But we all know the only valid entry name here is the JavaSE-1.8.

The bottom line is: the extension is trying to work with something that will never be accepted by VSCode.

Thank you for your responsiveness.

> the problem is that "name" should be JavaSE-1.8 instead of 8, right?
